wcc-contentful/lib/wcc/contentful/simple_client/response.rb

```diff
@@ -129,26 +131,31 @@ def next_page
 
       next_page = SyncResponse.new(next_page)
       next_page.assert_ok!
+      @next_page = next_page if @memoize
```

Comment:
I'm guessing this allows the old functionality if they use the `sync_old` method?

Reply:
Not if they use the method directly, but if they don't provide a `block`

Reply:
Correct